src/main/java/frc/robot/commands/gpm/IntakeNote.java

```diff
@@ -12,25 +12,32 @@ public class IntakeNote extends Command{
 
     private final Intake intake;
     private final StorageIndex storageIndex;
-    private final Arm arm;
+    //private final Arm arm;
```

Comment:
The arm should still be here. You can add a second constructor without the arm, and set it to null. In initialize and end, you can add a check for the arm being null before setting its angle.